client/main/host.js

```diff
@@ -42,6 +43,7 @@ let showOpenDialog = doNothing;
 let openUrl = doNothing;
 let openRecorder = doNothing;
 let triggerDownload = doNothing;
+let setBackstage = (obj) => { backstage = obj; };
```

Comment:
i think that something more generic might be in order here. this places a hard dependency on backstage. something like?:

```
setDialogProvider()
```